### PR TITLE
Unify labeling

### PR DESCRIPTION
**Pull request by alejoe91**
- labeling TP spikes now uses the same logic as counting matches. For large delta_frames (several ms) there was a mismatch in the numbers

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Not happy at all with the last commits.
We talk to leave a threhold for chance level !!!!!!
You remove it.
:(

**Comment by alejoe91** [comment]
That was a bad chance level! I'll use 0.1

**Comment by alejoe91** [comment]
@samuelgarcia this should be better :)

- now the BaseComparisons have `chanche_score` (0.1) and `match_score` (0.5)
- the GTComaprison also has: `well_detected_score` (0.8) and `redundant_score` (0.2)
- I propose to only base well-detected units on the score (as redundant and bad). Users can always check other metrics but I think that it's better to be simple and crystal clear

Agree?

**Comment by samuelgarcia** [comment]
Hi Alessio.
I will have look very soon this morning.

File: spikecomparison/comparisontools.py [hunk 1]
```diff
@@ -405,13 +405,24 @@ def do_score_labels(sorting1, sorting2, delta_frames, unit_map12, label_misclass
...
+            membership_sorted = membership[indices]
+            diffs = times_concat_sorted[1:] - times_concat_sorted[:-1]
+            inds = np.where((diffs <= delta_frames) & (membership_sorted[:-1] != membership_sorted[1:]))[0]
+            if len(inds) > 0:
+                inds2 = inds[np.where(inds[:-1] + 1 != inds[1:])[0]] + 1
+                inds2 = np.concatenate((inds2, [inds[-1]]))
+                times_matched = times_concat_sorted[inds2]
+                # find and label closest spikes
+                ind_st1 = np.array([np.abs(sts1[u1].tolist() - tm).argmin() for tm in times_matched])
+                ind_st2 = np.array([np.abs(mapped_st.tolist() - tm).argmin() for tm in times_matched])
```
Comment (samuelgarcia) [inline spikecomparison/comparisontools.py:421]: I still don't understand this tolist.

Comment (alejoe91) [reply spikecomparison/comparisontools.py:421]: me neither..but without it the spykingcircus comparison complains..could it be that SC spiketrains are stored differently?

**Comment by samuelgarcia** [comment]
@alejoe91 : I am happy with this.
I will merge it now.
Give me 2 hours to play with this code and possibly make some ultimate change in print_summary and study related to this before the release.

**Pull request merged**